### PR TITLE
Remodeling

### PR DESCRIPTION
**Pull request by NilsHullegien**
Added world modules and a seperate path.

### CODE REVIEW

**Comment by dVoorhout** [comment]
levelpieces don't line up perfectly with each other, I noticed a little gap between them

**Comment by NilsHullegien** [comment]
Whoops, I will fix this now, closing the PR.

**Comment by NilsHullegien** [comment]
Fixed the gap, reopening the PR.

File: ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Commander.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
     //String of the path to the commander model
     private static final String PATHNAME = "Models/ninja.j3o";
-    private static final Vector3f MOVE_VECTOR = new Vector3f(-0.2f, 0, 0);
+    private static final Vector3f MOVE_VECTOR = new Vector3f(-0.5f, 0, 0);
```
Comment (marcelkuipers) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Commander.java:23]: The commander now moves faster than the platform.

Comment (NilsHullegien) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Commander.java:23]: Forgot to change that back, changed it earlier for testing purposes. 
I will close the PR to fix this.

Comment (NilsHullegien) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Commander.java:23]: Fixed the derp, reopening PR.

**Comment by marcelkuipers** [comment]
Looks good now!

**Comment by NilsHullegien** [comment]
Closing PR to change models to fix the overlapping issues

**Comment by NilsHullegien** [comment]
Temporarily fixed the overlapping issues, the models will be reworked in another PR.

**Comment by dVoorhout** [comment]
overlapping will have a higher priority later on, for now things look fine

**Pull request merged**